### DISCUSSION TITLE
Dead links to download Delay architect.

### DISCUSSION THREAD

**Discussion by ericsoho** [jpcima/DelayArchitect#39] [Q&A]
Hello,
First of all congratulations for this plugin, it's my go to delay.
I talk about it on some social networks but the links to download it are dead.
Do you plan to reactivate them ?
If not, do you give me the authorisation to link my file (windows version) please ?
Best regards. Eric

PS : I'm french so if you speak french too that will be easier to communicate.

**Comment by diwowo** [comment]
Confirmed. Neither of the download links works so far.
